tests/lib/parse.js

```diff
@@ -105,5 +105,19 @@ describe("parse()", () => {
             );
         });
 
+        // https://github.com/eslint/espree/issues/470
+        it("Should throw on invaid `(a = 1) = t`", () => {
+            assert.throws(() => {
+                espree.parse("(a = 1) = t");
+            });
```

Comment:
This test is already passing with the default `ecmaVersion: 5` and Acorn 7. We should set a higher `ecmaVersion` to confirm that https://github.com/eslint/espree/issues/470 has been fixed.